helm/kfp-operator/templates/eventing/internal_eventbus.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if gt (len .Values.providers) 0 }}
+{{- if .Values.statusFeedback.enabled }}
```

Comment:
not a fan but again this is temporary till probably the public eventbus moves to provider helm and status updater replaced with webhook. This was just to keep feature same as existing